js/src/css/options.js

```diff
@@ -38,6 +38,11 @@ function Options(options) {
   var space_around_selector_separator = this._get_boolean('space_around_selector_separator');
   this.space_around_combinator = this._get_boolean('space_around_combinator') || space_around_selector_separator;
 
+  var brace_style_split = this._get_selection_list('brace_style', ['collapse', 'expand']);
```

Comment:
You'll need to accept the other values that are allowed by the javascript options, otherwise you'll get an error if someone passes them (via inheriting the option). You can ignore or convert them loop below.

Reply:
👍 done. Linked two more related issues as well